stdlib/REPL/src/docview.jl

```diff
@@ -574,12 +574,7 @@ print_correction(word) = print_correction(stdout, word)
 
 # Completion data
 
-const builtins = ["abstract type", "baremodule", "begin", "break",
-                  "catch", "ccall", "const", "continue", "do", "else",
-                  "elseif", "end", "export", "finally", "for", "function",
-                  "global", "if", "import", "let",
-                  "local", "macro", "module", "mutable struct", "primitive type",
-                  "quote", "return", "struct", "try", "using", "while"]
+const builtins = collect(keys(Base.Docs.keywords))
```

Comment:
```suggestion
```
and just put `collect(keys(Base.Docs.keywords))` directly on L586

Reply:
This seems much more efficient. Thank you!!